docs/integrations/channels/quick-start.mdx

```diff
@@ -29,14 +29,8 @@ Accept: application/json
   "platform": "drupal",
   "external_id": "",
   "is_enabled": true,
-  "status": "connected",
```

Comment:
There is a section above that might need to be rewritten a bit in context to this change:
>Apps curated as solutions within the **Create Channel** flow must create a basic channel with a reference to their BigCommerce [App ID](/docs/integrations/apps/guide/id). This enables them to be shown in the [Channel Manager](/docs/integrations/channels) as managing a specific channel; it also improves the native look and feel for the merchant using the app.